docs/examples.md

```diff
@@ -0,0 +1,170 @@
+# Examples
+
+- [Examples](#examples)
+  - [Creating a Wagtail Block for specific HTML structures](#creating-a-wagtail-block-for-specific-html-structures)
+  - [How to implement a handler for a specific HTML structure](#how-to-implement-a-handler-for-a-specific-html-structure)
+    - [Add extra configuration to allow this HTML tag is pass through bleach_clean()](#add-extra-configuration-to-allow-this-html-tag-is-pass-through-bleach_clean)
```

Comment:
**Suggestion**: Correct this typo "tag is pass through", here and at the heading itself. I'm not sure what was originally intended.